oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/authentication/OAuth2AuthorizationCodeAuthenticationProvider.java

```diff
@@ -83,4 +83,8 @@ public Authentication authenticate(Authentication authentication) throws Authent
 	public boolean supports(Class<?> authentication) {
 		return OAuth2AuthorizationCodeAuthenticationToken.class.isAssignableFrom(authentication);
 	}
+
+	public OAuth2AccessTokenResponse getAccessTokenResponse(OAuth2AuthorizationCodeGrantRequest oAuth2AuthorizationCodeGrantRequest) {
```

Comment:
This change cannot go into a patch release as this is targeted for 5.1.x. Please review the [acceptable changes](https://apr.apache.org/versioning.html) for patch, minor and major.